docs/content/mongocxx-v3/installation/_index.md

```diff
@@ -35,6 +35,93 @@ To configure and install the driver, follow the instructions for your platform:
 The Mongo C++ driver is available in the following package managers.
 - [Vcpkg](https://vcpkg.io/) (search for mongo-cxx-driver)
 - [Conan](https://conan.io/center/recipes/mongo-cxx-driver)
+- [Homebrew](https://formulae.brew.sh/formula/mongo-cxx-driver)
+
+### Vcpkg Install Instructions
+
+If you do not already have Vcpkg installed, install it with the following
+command:
+```
+$ git clone https://github.com/Microsoft/vcpkg.git
+$ cd vcpkg
+$ ./bootstrap-vcpkg.sh
+```
+
+Optionally, to install with Visual Studio integration:
+```
+vcpkg integrate install
+```
+
+Install the driver. (You may need to `git pull` to get the latest version of the
+driver)
+```
+$ ./vcpkg install mongo-cxx-driver
+```
+
+You can use the toolchain file, `vcpkg.cmake`, to instruct CMake where to find
+the development files, for example:
+```
+-DCMAKE_TOOLCHAIN_FILE=/<path to vcpkg repo>/vcpkg/scripts/buildsystems/vcpkg.cmake
+```
+
+You can find the header files in:
+```
+vcpkg/installed/<CPU ARCHITECTURE>-<OPERATING SYSTEM>/include/
+```
+
+The library files are in:
+```
+vcpkg/installed/<CPU ARCHITECTURE>-<OPERATING SYSTEM>/lib/
+```
+
+### Conan Install Instructions
+
+Package Specifier: `mongo-cxx-driver/3.8.0`
+
+If you do not already have Conan installed, then install it and run the Conan
+initalization command below:
+```
+$ pip install conan
+$ conan profile detect --force
+```
+
+Add the following to your `conanfile.txt`
+```
+[requires]
+mongo-cxx-driver/3.8.0
+[generators]
+CMakeDeps
+CMakeToolchain
+```
+
+Install the driver via Conan, and build your project:
+```
+$ conan install conanfile.txt --output-folder=build --build=missing
+$ cmake \
+	-B build \
+	-DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake \
+	-DCMAKE_BUILD_TYPE=Release
+$ cmake --build build
+```
+
+### Homebrew
+
+For MacOS users, homebrew is a convienent way to install the C++ driver.
+
+```
+brew install mongo-cxx-driver
+```
+
+Headers can be found in:
+```
+/opt/homebrew/include/mongocxx/v_noabi/
+/opt/homebrew/include/bsoncxx/v_noabi/
+```
+
+Library files can be found in:
+```
+/opt/homebrew/lib/
```

Comment:
This is true only for Apple Silicon. For intel based Macs, the installation happens with ` /usr/local` prefix. Good to add both cases. 
Reference - https://docs.brew.sh/Installation

Reply:
I just pushed an update for Intel Mac instructions. I do not have access to an Intel based Mac, does anyone else have access to an Intel Mac who would be able to verify?

Reply:
Thanks @adriandole for verifying that this is the case!